File: iOS/Matcha/Matcha/Sources/TestCase.swift
//
//  Feature.swift
//  Vicky
//
//  Created by Andre Guggenberger on 10/05/2017.
//  Copyright © 2017 CASEapps. All rights reserved.
//

import Foundation

open class TestCase {
    var fileName : String
    var name : String
    var scenarios:[Scenario] = []
    
    init(fileName: String, name: String) {
        self.fileName = fileName
        self.name = name
    }
}

File: Android/Matcha/matchalib/src/main/java/at/caseapps/matcha/TestCase.java
package at.caseapps.matcha;

import java.util.ArrayList;
import java.util.List;

public class TestCase {
    public String fileName;
    public String name;

    public List<Scenario> scenarios = new ArrayList<>();

    public TestCase(String fileName, String name) {
        this.fileName = fileName;
        this.name = name;
    }
}
